docs/design/007-cli-payments/README.md

```diff
@@ -154,4 +154,33 @@ All commands to send domain transactions require new optional params to add tran
 [fees_inputs=<source-1>,..,<source-n>] [fees_outputs=(<recipient-0>,<amount>),..,(<recipient-n>,<amount>)] [extra=<extra>]
 ```
 
-Note that "source-n" is identifier presented in "Source" column of ```ledger get-sources``` command output
\ No newline at end of file
+Note that "source-n" is identifier presented in "Source" column of ```ledger get-sources``` command output
+
+### Sign with a payment address
+
+Many cryptocurrencies provide mechanisms to prove control of a given payment address.
+The following command signs messages using the private key that corresponds to the specified payment address.
+
+```indy-cli
+indy> payment-address sign address=<payment-address> input=<input>
+```
+
+"address" is the payment address generated from calling `create`.
+
+"input is the message to be signed.
+
+The output is a hex encoded signature that can be passed to verify
+
+### Verify signature from a payment address
+
+Verifies a proof of payment address control by verifying a signature.
+
+```indy-cli
+indy> payment-address verify address=<payment-address> input=<input> signature=<signature>
+```
+
+"address" is the payment address generated from calling `create`. This should be the same address used to create "signature"
+
+"input is the message that was signed. This must be the same message used to create the "signature" otherwise verify will return false.
```

Comment:
```suggestion
"input" is the message that was signed. This must be the same message used to create the "signature" otherwise verify will return false.
```